src/Creitive/Breadcrumbs/BreadcrumbsServiceProvider.php

```diff
@@ -11,7 +13,14 @@ class BreadcrumbsServiceProvider extends ServiceProvider {
 	 */
 	public function boot()
 	{
-		$this->package('creitive/breadcrumbs');
+        if ($this->isLegacyLaravel() || $this->isOldLaravel())
```

Comment:
I'm not sure I'm comfortable with code (not) executing based on the Laravel version it's running in. I've done this before, but I feel that there are better ways to solve this problem.